src/PTFAnalysis.cpp

```diff
@@ -698,7 +700,9 @@ PTFAnalysis::PTFAnalysis( TFile* outfile, Wrapper & wrapper, double errorbar, PT
       }else{
         fitresult->numPulses = 0;
       }
-        
+	  if( config.Get("do_pulse_fitting", dofit) ){
+	         dofit=true ;
```

Comment:
Again, this seems wrong.  You are saying that as long as 'do_pulse_fitting' is in the config.dat file you will do pulse fitting, rather than using the value of 'do_pulse_fitting' in the config file.  This conditional check should just be deleted.

Reply:
You are absolutely right, I was now understanding how to properly set these variables. I made a fix that hopefully solves all of this,I had to manually insert a line after these statement since it seemed like it was getting the value (true or false) of the fitting only in the beginning.